cgv_clone/src/main/java/com/ceos22/cgv_clone/domain/orderFood/Food.java

```diff
@@ -1,4 +1,40 @@
 package com.ceos22.cgv_clone.domain.orderFood;
 
+import com.ceos22.cgv_clone.exception.NotEnoughStockException;
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "food")
+@Getter @NoArgsConstructor(access = AccessLevel.PROTECTED)
 public class Food {
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String name;
+    private int price;
+    private int stockQuantity;
+
+    //==생성 메서드==//
+    public Food(String name, int price, int stockQuantity) {
+        this.name = name;
+        this.price = price;
+        this.stockQuantity = stockQuantity;
+    }
+
```

Comment:
p5. 생성자 라고 부르죠 ㅎㅎ

Intellij 를 사용하고 계신다면 아래와 같은 주석 기능이 있습니다

```suggestion
    //region 생성자
    public Food(String name, int price, int stockQuantity) {
        this.name = name;
        this.price = price;
        this.stockQuantity = stockQuantity;
    }
    //endregion
```